redaxo/src/addons/media_manager/lib/effects/effect_convert2img.php

```diff
@@ -136,6 +146,9 @@ private function getConvertPath()
                 $path = $out[0];
             }
         }
+		if (extension_loaded('imagick')) {
+			$path = 'imagick';
```

Comment:
das macht hier unten keinen sinn, sollte oben in dem anderen teil der änderung gesteuert werden.

Reply:
Macht schon Sinn, da sonst die Meldung im Effekt erscheint, dass exec nicht verfügbar ist.

Reply:
rein von der steuerung her macht es sinn. es ist hier aber die falsche stelle

Reply:
Änderungen kannst Du gern vornehmen.
Es wäre halt schön, wenn nicht nur exec genutzt werden würde.